aicsimageio/utils/io_utils.py

```diff
@@ -44,7 +48,7 @@ def pathlike_to_fs(
         uri = str(uri)
 
     # Get details
-    fs, path = url_to_fs(uri)
+    fs, path = url_to_fs(uri, **fs_kwargs)
```

Comment:
All to get to this line, right?
I almost would rather let users pass in their own fs object that they can create with our pathlike_to_fs, for this reason:  passing `anon=False` is so specific to the s3fs that it would be more readable code if the user passed `sf3s(anon=False)` instead. 
How do other libs handle these specific fs args?

Reply:
Actually this handles all fs's: https://filesystem-spec.readthedocs.io/en/latest/api.html?highlight=url_to_fs#fsspec.core.url_to_fs

We should definitely allow more URI handling patterns but for now its a start imo.

Reply:
I'm just saying that this:
`img = AICSImage("s3://my-bucket/my_file.tiff", fs_kwargs=dict(anon=True))`
looks uglier than this:
`img = AICSImage("s3://my-bucket/my_file.tiff", fs=s3fs(anon=True))`
I appreciate that we are trying to do it "automatically" but it's not very automatic when you have to explicitly pass in that anon value.